starter/src/test/java/com/starter/fullstack/dao/InventoryDAOTest.java

```diff
@@ -47,6 +47,7 @@ public void createTest() {
     System.out.print("The initial ID is: " + inventory1.getId());
     this.inventoryDAO.create(inventory1);
     System.out.println("After Create Method ID: " + inventory1.getId());
+    Assert.assertTrue(inventory1.getId() != "877241");
```

Comment:
This is part of the reason you should use the variables you have defined at the class level. Although the contents of the strings are the same, they are technically two different strings. You should use a variable in both the creation of the item and the checking of the item.